recipe/meta.yaml

```diff
@@ -21,8 +21,9 @@ requirements:
     - pip
   run:
     - python >=3.7,<4
-    - rich >=12.3,<13
+    - rich >=12.6,<13
```

Comment:
If you add `- importlib-metadata >=4.11.3,<5`, it should pass

Reply:
thanks @beenje, hopefully now it should go. Sorry I didn't have the time to look into this.